dynamicio/io/postgres.py

```diff
@@ -130,32 +94,23 @@ def read(self) -> pd.DataFrame:
         else:
             sql_query = self.sql_query
 
-        logger.info(f"Downloading table: {self.final_table_name} from: {self.db_host}:{self.db_name}")
+        logging.info(f"Downloading table: {self.final_table_name} from: {self.db_host}:{self.db_name}")
         with session_scope(self.connection_string, self.application_name) as session:
-            df = pd.read_sql(sql=sql_query, con=session.get_bind(), **self.kwargs)
-
-        df = self.validate(df)
+            df = pd.read_sql(sql=sql_query, con=session.get_bind(), **self.read_kwargs)
 
```

Comment:
We don't want to validate anymore for this resource?

Reply:
Validation happens in `resource.py`.